### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by ThunderGemios10** [ThunderGemios10/Survival-Games#20]
So, I think it's about time we start documenting the plugin @sshipway

We can use the wiki feature :)
https://github.com/ThunderGemios10/Survival-Games/wiki

**Comment by sshipway** [comment]
Good idea, there are several things that are not immediately obvious.  Chest levels and bandages for a start...

Now all I need to do is to find some free time...
